app/javascript/tests/close_contact/CloseContact.test.js

```diff
@@ -0,0 +1,185 @@
+        emptyCCWrapper.find(Modal.Body).find('Row').at(index).find(iv.inputType).simulate('change', iv.value)
+        expect(handleDateChangeSpy).toHaveBeenCalled();
+      } else {
+        emptyCCWrapper.find(Modal.Body).find('Row').at(index).find(iv.inputType).simulate('change', { target: { id: iv.field, value: iv.value } })
+        expect(handleChangeSpy).toHaveBeenCalled();
+      }
+      expect(emptyCCWrapper.state(iv.field)).toEqual(iv.value);
+    })
+  })
+
+  it('Properly creates a the correct assigned user dropdown options', () => {
+    const emptyCCWrapper = getShallowWrapper(mockCloseContact0);
+    emptyCCWrapper.find(Button).simulate('click');
+    ASSIGNED_USERS.forEach((user, userIndex) => {
+      expect(emptyCCWrapper.find(Modal.Body).find('option').at(userIndex).text()).toEqual(`${user}`);
+    })
+  });
+
+  it('Properly clears any values in the modal on close', () => {
+    const emptyCCWrapper = getShallowWrapper(mockCloseContact0);
+    expect(emptyCCWrapper.state('showModal')).toBeFalsy();
+    emptyCCWrapper.find(Button).simulate('click');
+    expect(emptyCCWrapper.state('showModal')).toBeTruthy();
+
+    // This iteration assumes the testInputValues order matches the order of the elements in the DOM
+    testInputValues.forEach((iv, index) => {
+      if (iv.inputType === 'DateInput') {
+        emptyCCWrapper.find(Modal.Body).find('Row').at(index).find(iv.inputType).simulate('change', iv.value)
+      } else {
+        emptyCCWrapper.find(Modal.Body).find('Row').at(index).find(iv.inputType).simulate('change', { target: { id: iv.field, value: iv.value } })
+      }
+    })
+
+    emptyCCWrapper.find(Button).at(1).simulate('click');
+    // Once the modal is closed, the values should default back to their nulled out original values
+    expect(emptyCCWrapper.state('showModal')).toBeFalsy();
+    expect(emptyCCWrapper.state('first_name')).toEqual(mockCloseContact0.first_name || '')
+    expect(emptyCCWrapper.state('last_name')).toEqual(mockCloseContact0.last_name || '')
+    expect(emptyCCWrapper.state('primary_telephone')).toEqual(mockCloseContact0.primary_telephone || '')
+    expect(emptyCCWrapper.state('email')).toEqual(mockCloseContact0.email || '')
+    expect(emptyCCWrapper.state('last_date_of_exposure')).toEqual(mockCloseContact0.last_date_of_exposure || null)
+    expect(emptyCCWrapper.state('assigned_user')).toEqual(mockCloseContact0.assigned_user || null)
+    expect(emptyCCWrapper.state('notes')).toEqual(mockCloseContact0.notes || '')
+  })
+
+  it('disables submit by default', () => {
+    const emptyCCWrapper = getShallowWrapper(mockCloseContact0);
+    expect(emptyCCWrapper.state('showModal')).toBeFalsy();
+    emptyCCWrapper.find(Button).simulate('click');
+    expect(emptyCCWrapper.state('showModal')).toBeTruthy();
+    expect(emptyCCWrapper.find(Button).at(2).props().disabled).toBeTruthy()
+  })
+
+  it('enables submit when first and last name are input', () => {
```

Comment:
'Enables submit button when first and last name are not empty'

Reply:
it looks like when you put in one or the other it enables the button, so I would update this test to do first name enables, clear out first name and check disabled, then test last name enabled